packages/apps/plugins/plugin-navtree/src/translations.ts

```diff
@@ -9,7 +9,12 @@ import { NAVTREE_PLUGIN } from './meta';
 export default [
   {
     'en-US': {
-      [NAVTREE_PLUGIN]: {},
+      [NAVTREE_PLUGIN]: {
+        'open settings label': 'Settings',
```

Comment:
i don't think these are intrinsically part of the navtree plugin though.
other apps might not use the navtree, which is why i moved them out (since i started with them here)... 
perhaps we should do what you've done with settings and create another plugin?
TODO?

Reply:
yeah this will likely go away, probably deserves a TODO. The sidebar will likely introduce surfaces or something like that. I think it's better here than in the `os` translations though, this at least couples the translation with the plugin that's using it.